word-search.py

```diff
@@ -5,6 +5,7 @@
 import shutil  #Module for getting terminal dimensions
 import random
 import re      
+from collections import defaultdict
```

Comment:
@vickydepasupil already moved the import library at line 8 :))

Reply:
Thank you!